eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/actions/AddToCalendarActionExecutable.java

```diff
@@ -72,7 +73,7 @@ private Bundle extrasForAddToCalendar()
         Bundle bundle = new Bundle(8);
         bundle.putLong(CalendarContract.EXTRA_EVENT_BEGIN_TIME, mEvent.start().getTimestamp());
         bundle.putLong(CalendarContract.EXTRA_EVENT_END_TIME,
-                mEvent.start().addDuration(mEvent.duration()).getTimestamp());
+                mEvent.start().addDuration(mEvent.duration().value(mEvent.start().isAllDay() ? new Duration(1, 1, 0) : Duration.parse("PT2H"))).getTimestamp());
```

Comment:
Could we have this as something like `FallbackDuration implements Lazy<Duration>`?:
```
mEvent.start().isAllDay() ? new Duration(1, 1, 0) : Duration.parse("PT2H")
```
So this fallback value definition is not that hidden and mingled with the calendar event setup.